sysvinit/netbsd/cloudconfig

```diff
@@ -10,6 +10,7 @@ name="cloudinit"
 start_cmd="start_cloud_init"
 start_cloud_init()
 {
+    kenv -q kernel_options | grep -q 'cloud-init=disabled' && exit 0
```

Comment:
this won't work on NetBSD. there's no kenv, and no way to pass or read out kernel options